docs/chapters/workflow/release/release-process.rst

```diff
@@ -32,6 +32,7 @@ List of artifacts
     - Instructions or pointers to instructions that have changed as a result of
       the changes since last release.
     - This list of artifacts.
+- The source code of all the packages released image contains.
```

Comment:
packages should be packages'

Reply:
Now when I read it again, I interpret the sentence differently. It seems like this,

The source code of all the packages, that released image contains.

So the issue is that the sentence can be interpreted in 2 ways. At least thats how I see it :)

Reply:
or it can also be written as,

The source code of all the packages, of the released image.

Reply:
okay, I got your point; I'll rephrase.

Reply:
iz it betta naw?